Makefile

```diff
@@ -97,15 +97,17 @@ include contrib/devtools/Makefile
 ###                                  Build                                  ###
 ###############################################################################
 
-build: go.sum
-	go build -mod=readonly ./...
+BUILD_TARGETS := build install
 
-simd:
-	mkdir -p $(BUILDDIR)
-	go build -mod=readonly $(BUILD_FLAGS) -o $(BUILDDIR) ./simapp/simd
+build: BUILD_ARGS=-o $(BUILDDIR)/
```

Comment:
please fix the docker image, this breaks it

Reply:
fixed